### PR TITLE
Fixed dynamic properties deprecation in PHP 8.2

### PR DESCRIPTION
**Pull request by meirzamoodle**

### CODE REVIEW

File: drivers/adodb-postgres64.inc.php [hunk 1]
```diff
@@ -91,6 +91,15 @@ class ADODB_postgres64 extends ADOConnection{
 	var $version;
 	var $_nestedSQL = false;
 
+	/** @var array an array with client, protocol and server values. */
+	var $version;
+
+	/** @var bool */
+	var $_nestedSQL = false;
```
Comment (matt-land) [inline drivers/adodb-postgres64.inc.php:98]: duplicate, declared on line 92

File: adodb-xmlschema03.inc.php [hunk 2]
```diff
@@ -1401,6 +1406,9 @@ class adoSchema {
 	 */
 	var $existingData;
 
+	/**  @var dbTable A table object. */
+	var $obj;
```
Comment (matt-land) [inline adodb-xmlschema03.inc.php:1410]: I didn't find any references to this in the class

File: drivers/adodb-sybase.inc.php [hunk 3]
```diff
@@ -47,6 +47,12 @@ class ADODB_sybase extends ADOConnection {
 
 	var $port;
 
+	/** @var mixed */
```
Comment (matt-land) [inline drivers/adodb-sybase.inc.php:50]: I only see this used as a bool (vs mixed), though its only read from and never set so I'm not sure

**Review by matt-land — CHANGES_REQUESTED** [review]
All comments that reference "databaseName" could also say that "databaseName" is deprecated, preferring the value "database"

The change to drivers/adodb-postgress64.inc.php is the most pressing.

File: drivers/adodb-postgres64.inc.php (hunk 1 above)
Comment (meirzamoodle) [reply drivers/adodb-postgres64.inc.php]: Hi @matt-land. I just updated it to remove those lines. Thanks.

**Review by matt-land — APPROVED** [review]

**Comment by dregad** [comment]
All issues raised in this PR have been addressed in separate commits.

**Comment by meirzamoodle** [comment]
> Out of curiosity, did you use some static analysis tool to detect and fix these issues ? If so it would be interesting to know which one.

Hi, @dregad. I'm using PHPStan to scan the all files.

**Comment by dregad** [comment]
Thanks for the feedback. Was it the pro version that proposed the fixes, or did you code them by hand ?

**Comment by meirzamoodle** [comment]
> Thanks for the feedback. Was it the pro version that proposed the fixes, or did you code them by hand ?

Sorry for the late reply. PHPStan is free.

**Pull request closed without merging**